recipes/gptools/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "gptools" %}
+{% set version = "0.2.3" %}
+{% set sha256 = " 412ccaf9bed17c00557e5a13b0e31b9841847fa0d3a9f823d0bbd73e0933aef9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
```

Comment:
I think only `python`, `setuptools`, `toolchain` and `numpy` are required to build the package. The rest look to be runtime requirements. 

Also see the following about building against numpy:
https://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy